### ISSUE TITLE
Understand Simple Galery file sorting??

### ISSUE THREAD

**Issue by DrSaad77** [SimpleMobileTools/Simple-Gallery#2510]
I can not understand, when I open a picture with Simple Galery, the next picture when I slide left or right, is never the same right or left in file Explorer.
Explanation: lets say, got picture called 1, 2 , 3, 4 , 5 ,6, 7, 8 in a folder. If I open picture 4, when I slide left should be picture 3, correct?? Right slide, should be pic 5, correct? But it is another story, it show example pic 9, then pic 2 , it is completly confusing! I could not understand how SimpleGalery is set to read ordering of those pics.
So solution, is to increase thumbnails in files explorer, and select then pic I wish to delete, totally without SimpleGalery use.

**Comment by tibbi** [comment]
you have to use sorting by file name for that. Every app has its own sorting, do not mix gallery and file managers

**Comment by DrSaad77** [comment]
Thx you, I know that each app has its own files sorting, and file sorting on SimpleGallery is fine and pretty rich, no problem there.
The thing is, I rarely use Simple Gallery directly, what I want to mean, is that I do not enter to Simple Gallery itself. 
I use File Manager , it is what I like with Android, you feel like on Windows,  I do a lot of file sorting and managing on my phone. So as I said, I go in my file Manager apps, and into some image/photo, and when I click on pic, Android asks which Viewer I want to use, right, and select SimpleGallery, and check that said pic in full screen, then when I slide left or right it is never the same sorting as the file Manager used. Of course, I made sure, to sort the files the same as SimpleGallery. In file Manager, files sorted by Name, and SimpleGallery also sorted by name. but it do not help.

**Comment by tibbi** [comment]
tell exact steps for reproducing the issue please. Also, what app version are you using?

**Comment by DrSaad77** [comment]
I am currently using version 6.22.2, because I was experiencing an issue similar to issue, after deleting a photo , it take me to first photo of the folder. I will try to record it issue right when it happens, for better understanding.
App I am using FX Explorer as file manager.

**Comment by tibbi** [comment]
that is an old app version, it has been fixed already

**Comment by DrSaad77** [comment]
I know that version 6.22.2.
I was using last version, and from last 3 or 4 versions, I am trying to tell you that I experienced a similar issue to #2508, when I was viewing some photo, and delete it, it takes me back to first picture of the folder. That is why I installed for now, an old version, it do not do that.
and second issue is current post : #2510
I will record a small video of the issues, there a 2.
